bdd/spec/006_string_lnn_spec.sh

```diff
@@ -4,31 +4,26 @@ Describe "Strings"
   Describe "most operations"
     before() {
       lnn_sourceToAll "
-        from @std/app import start, stdout, exit
+        from @std/app import start, print, exit
 
         on start {
-          emit stdout concat('Hello, ', \"World!\n\");
-          // concat('Hello, ', 'World!').print();
-          // print('Hello, ' + 'World!');
+          concat('Hello, ', \"World!\").print();
+          print('Hello, ' + 'World!');
 
-          emit stdout concat(repeat('hi ', 5), \"\n\");
-          // repeat('hi ', 5).print();
-          // print('hi ' * 5);
+          repeat('hi ', 5).print();
+          print('hey ' * 5);
 
-          emit stdout concat(toString(matches('foobar', 'fo.*')), \"\n\");
-          // matches('foobar', 'fo.*').print();
-          // print('foobar' ~ 'fo.*');
+          matches('foobar', 'fo.*').print();
+          print('foobar' ~ 'fo.*');
 
           // index('foobar', 'ba').print();
-          // print('foobar' @ 'ba');
+          // print('foobar' @ 'ra');
```

Comment:
I know this is still commented out, but is there a reason for choosing an invalid string here? To confirm the error message in the future, I guess?

Reply:
it's to test the behavior of `index` to make sure that it doesn't false-positive on this input (i didn't have to do it in this PR, but i thought of it during my refactors and thought it'd be good to do)